config/percep/config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "frame_write_interval": 10,
+    "buffer_iterations": 20,
+    "DEFAULT_TAG_VAL": -1,
+    "mm_per_m":1000,
+    "threshold_confidence":90,
+
+    "pt_cloud":
+    {   
+        "max_field_of_view_angle": 70,
+        "pt_cloud_width": 320,
+        "pt_cloud_height": 180,
+        "rover_w_mm": 1168,
+        "half_rover": 584,
+        "centerx": 0,
+        "downsample_voxel_filter": 20.0,
+       
+        "ransac": {
+            "max_iterations": 400,
+            "segmentation_epsilon": 10,
+            "distance_threshold": 100
+        },
+
+        "pass_through": {
+            "upperBdZ": 7000.0,
```

Comment:
NIT: what does the switch from snake to camel case signify? If there is no difference then use only snake or only camel case